mcs/class/Mono.Security/Test/Mono.Security.Authenticode/Formatter/Makefile

```diff
@@ -0,0 +1,12 @@
+NONO_PATH ?= /inst/m6/lib
+
+all: writefiles hashfile.exe
+
+clean:
+	-rm writefiles hashfile.exe
+
+writefiles: writefiles.cpp
+	g++ writefiles.cpp -o $@
+
+hashfile.exe: hashfile.cs
+	mcs hashfile.cs /r:$(NONO_PATH)/mono/4.7.1-api/Mono.Security.dll
```

Comment:
The path is odd it should be something like ../../lib/$(PROFILE)/Mono.Security.dll if you move it to Mono.Security Makefile